janitor/functions.py

```diff
@@ -1444,7 +1442,7 @@ def remove_columns(
 
     .. code-block:: python
 
-        df = pd.DataFrame(...).remove_columns(column_names=['col1', ['col2']])
+        df = pd.DataFrame(...).remove_columns(column_names=['col1', 'col2'])
```

Comment:
It seemed like the extra set of square braces here was unintentional and might not work as expected.